CONTRIBUTING.md

```diff
@@ -4,10 +4,10 @@ Contributing is easy!
 
 1. [Fork this repository](https://github.com/atom/flight-manual.atom.io#fork-destination-box)
 2. Download your fork. [(Click here for help)](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced)
-3. Create a named feature branch ([we use Github flow](https://guides.github.com/introduction/flow/index.html))
+3. Create a named feature branch [(we use Github flow)](https://guides.github.com/introduction/flow/index.html)
```

Comment:
This has the effect of making the parentheses part of the link as well.  I don't like that, and think that the one above this should also be changed so that the parentheses are excluded from the link.